spring-boot-admin-server-ui/src/main/frontend/services/instance.js

```diff
@@ -179,6 +179,19 @@ class Instance {
     return this.axios.get(uri`actuator/threaddump`);
   }
 
+  async downloadThreaddump(instanceName) {
```

Comment:
There is no need to pass in the instance name. It is available as `this.registration.name`